setup.py

```diff
@@ -20,7 +20,7 @@ def read(fname):
         'termcolor>=1.1.0,<2',
         'openstacksdk>=0.10.0,<1',
         'spur>=0.3.20,<1',
-        'kubernetes>=1.0.0,<2',
+        'kubernetes>=8.0.1<9',
```

Comment:
should probably go to 9.0.x since 1.13 is current and 1.14 is coming very soon.

Reply:
+1